### PR TITLE
Update CloudStack API metadata

### PR DESCRIPTION
**Pull request by olivierlemasle**

### CODE REVIEW

File: apis/cloudstack/src/main/java/org/jclouds/cloudstack/CloudStackApiMetadata.java [hunk 1]
```diff
@@ -62,12 +62,12 @@ public static Properties defaultProperties() {
...
          id("cloudstack")
-         .name("Citrix CloudStack API")
+         .name("Apache CloudStack API")
          .identityName("API Key")
          .credentialName("Secret Key")
-         .documentation(URI.create("http://download.cloud.com/releases/2.2.0/api_2.2.12/TOC_User.html"))
+         .documentation(URI.create("https://cloudstack.apache.org/api/apidocs-4.5/TOC_Root_Admin.html"))
          .defaultEndpoint("http://localhost:8080/client/api")
-         .version("2.2")
+         .version("4.5")
```
Comment (nacx) [inline apis/cloudstack/src/main/java/org/jclouds/cloudstack/CloudStackApiMetadata.java]: This field defines the default CloudStack version supported in jclouds. Have you been able to verify that the current code is actually compatible with version 4.5? can you share the output of the live tests?

**Comment by andrewgaul** [comment]
@olivergondza do we have a path forward on this pull request?

**Comment by stevegomez17** [comment]
your decision

**Comment by andrewgaul** [comment]
Closed due to inactivity.

**Pull request closed without merging**